docs/cloud/index.md

```diff
@@ -1,168 +1,49 @@
-process on a host  machine. It exposes a part of  the resources of the
-underlying machine to a guest  operating system. This approach is more
-flexible compared  to using just  physical servers, as a  single large
-physical  server  can be  partitioned  into  isolated smaller  virtual
-machines.
-
-#### Virtual network
-
-One  or   more  virtual  networks   can  run  on  the   same  physical
-network. This  helps prevent  users from  seeing each  others' network
-traffic in a shared infrastructure.  In IaaS services virtual networks
-are assigned to specific users or  groups of users to provide security
-and ensure fair use of shared network links.
-
-#### Virtual machine image
-
-A large  file containing a  complete state  of a virtual  machine. The
-majority of a virtual machine image  is the simulated hard disk of the
-virtual machine. As such virtual machine images vary in size depending
-on  the  configuration  of  the  virtual  machine  and  the  installed
-applications.   Virtual  machine images  are  the  starting point  for
-creating  new  virtual  machines  and   they  can  be  easily  cloned,
-snapshotted, backed up and managed.
-
-  [cloud-stack]: /img/cloud-stack.png
+# Overview
+
+CSC offers a variety of cloud computing services: Pouta services (cPouta and ePouta) and Rahti container cloud service.
+
+Pouta services are provided in a Infrastructure-as-a-Service model. Using Pouta services, you can can take control of your IT infrastructure, manage it in a flexible fashion, and automate it as you require. You can launch and manage virtual machines, deploy storage, and create networks for your needs. cPouta is targeted towards generic cloud needs, while ePouta is mainly used for handling sensitive data. Taking ePouta into use requires more set-up and always involves your organization's IT department.
+
+Rahti makes the deployment of applications a breeze. This is a higher level service than Pouta services, and Rahti automatically takes care of a lot of the lower level management, so that you don't have to. Rahti is based on Red Hat's OKD, which is a distribution of Kubernetes. It uses Docker containers to run applications.[^1]
+
+### Pouta
+
+With Pouta services, you can can take control of your IT infrastructure, manage it in a flexible fashion, and automate it as you require. You can launch and manage virtual machines, deploy storage, and create networks for your needs. This infrastructure can be used for anything you need or can imagine.[^2]
+
+<iframe
+    width="400"
+    height="300"
+    srcdoc="https://www.youtube.com/embed/2opuVIBSPgw"
+    title="CSC's Pouta Cloud Services"
+    frameborder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media;gyroscope; picture-in-picture"
+    allowfullscreen>
+</iframe>
+
+More information about using Pouta may be found from [here](pouta/).
+
+### Pukki - Database as a Service
+
+Something about Pukki - Database as a Service.
```

Comment:
Suggestion:

Pukki is a that provides on demand databases with automatic backups and maintenance. If you have a CSC-account you should be able to get your first database within 15 minutes and your second database within five. If you are doing research, arranging courses or building a new services that needs databases you probably want to check out this service before you start setting up manually your own database.